_projects/2024/100540514.Rmd

```diff
@@ -0,0 +1,554 @@
+
+![](images/Orig Graph.png){width="100%"}
+
+## Getting the Data
+
+#The data for this graph was derived from the SARS-CoV-2 genomic surveillance efforts coordinated through the COVID-19 Genomic Surveillance Regional Network (COVIGEN). From its inception in March 2020 to March 2022, COVIGEN supported the sequencing of 126,985 SARS-CoV-2 genomes from 32 countries and territories in Latin America and the Caribbean (LAC). To create a more comprehensive dataset, these genomes were supplemented with additional sequences and metadata obtained from the EpiCoV database hosted on GISAID, covering the same period and geographic region. This aggregation resulted in a dataset of 296,286 genomes from 41 countries and territories, providing a summarized view of genomic diversity and sequencing efforts across the region. The data was further aggregated by combining geographically proximate countries into broader locations, enabling a summarized representation of viral sequencing and lineage diversity. The data specific to this graph was available in the paper's "Data Availability" section.
+
+## Recreating the Graph
+### Required Libraries
+
+- `tidyverse`: unifies workflows by integrating ggplot2 for graphics and dplyr for data manipulation, providing a consistent and intuitive grammar for transforming and visualizing data.
+- `dplyr`: allows intuitive manipulation of data, readable operations on datasets through verb-like functions that transform and reshape data in predictable ways.
+- `ggtext`: an extension to ggplot2 that enhances text rendering capabilities
+- `googlesheets4`: reads and writes Google Sheets directly in R
+
+
+```{r}
+library(tidyverse)
+library(dplyr)
+library(ggtext)
+library(googlesheets4)
+```
+
+### Country Order Definition
+#This step defines a custom order for the countries in the visualization. The order is crucial for the graph's readability
+
+```{r}
+country_order <- c(
+  "Argentina", "Bolivia", "Brazil", "Chile", "Colombia", "Ecuador", "Guyana",
+  "Paraguay", "Peru", "Suriname", "Uruguay", "Venezuela", "Honduras",
+  "Guatemala", "El Salvador", "Costa Rica", "Panama", "Belize", "Nicaragua",
+  "Mexico", "Bahamas", "Bermuda", "Cuba", "Dominican Republic", "Haiti",
+  "Jamaica", "Barbados", "Dominica", "Saint Barthélemy", "Grenada",
+  "Saint Lucia", "Guadeloupe", "Saint Kitts and Nevis", "Trinidad and Tobago",
+  "Saint Martin", "Saint Vincent and the Grenadines", "Antigua and Barbuda",
+  "Curacao", "Aruba", "Bonaire", "Sint Maarten"
+)
+```
+
+### Data Loading Preparation
+Use read_sheet() to load data from Google Sheets
+Convert Country to a factor with a specific order
+Remove any rows with missing country data
+
+```{r}
+covid_data <- read_sheet("https://docs.google.com/spreadsheets/d/18HUp5w-ncftAzsfPpR9yiIo-IS7ui_QmjGEd0O-LGWk/edit?gid=0#gid=0") %>%
```

Comment:
How big is this file? If it's reasonable, just download it, put it close to the Rmd and add it to this PR. Then read the file directly. If it's huge, then at least make this link public, because access is restricted!